packages/sdk/src/oracle/optimistic/client.ts

```diff
@@ -1,28 +1,31 @@
 import assert from "assert";
 import { ethers } from "ethers";
-import Store, { Emit } from "./store";
+
```

Comment:
this is the original client, it should function the same, its just been changed to use the new interfaces and plugin the necessary handlers at run time